source/jormungandr/jormungandr/street_network/kraken.py

```diff
@@ -103,6 +104,8 @@ def _direct_path(
             'max_car_no_park_duration_to_pt',
             'taxi_speed',
             'max_taxi_duration_to_pt',
+            'ridesharing_speed',
```

Comment:
why added now the ridesharing parameters ?

Reply:
because `ridesharing_speed` was stored in `car_no_park_speed` before this PR